lib/bolt/config.rb

```diff
@@ -228,6 +228,11 @@ def normalize_overrides(options)
       # Set console log to debug if in debug mode
       if options[:debug]
         overrides['log'] = { 'console' => { 'level' => :debug } }
+      elsif options[:verbose]
```

Comment:
Logging in Bolt still seems to be a little messy. I think this is more correct than what I had before, but made sure I wouldn't overwrite an existing level. And if console logging gets more options, both debug and verbose need to be fixed here.